### DISCUSSION TITLE
Welcome to dmidiplayer Discussions!

### DISCUSSION THREAD

**Discussion by pedrolcl** [pedrolcl/dmidiplayer#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by practicafz** [comment]
Hi Pedro! You know me because I had a website with a Midi Player I coded in Delphi many years ago. I was advertising your player as a good replacement for it on the site. I just took down that website today, but I still have a music website, with lots of MIDI files, and I will rework an ad for your software into that site, shortly.

I like what you are doing. If I was about 30 years younger, I might be tempted to dig into programming again, and help you with this. As it is, I can only offer comments and suggestions from the "peanut gallery". Best wishes!

**Reply by pedrolcl** [reply]
Hi Frank! I'm glad to read you! Thank you for coming! Your issue reports and comments are quite valuable contributions.

**Comment by practicafz** [comment]
As per my blog comment, what does "File->Song Settings->Save" actually do?

**Reply by pedrolcl** [reply]
It is documented in both [the web site](https://dmidiplayer.sourceforge.io/manual.shtml#song-settings) and the installed internal program documentation: Help menu->Contents->Song Settings.

In Windows, the location of the song configuration files is the directory `%USERPROFILE%\.dmidiplayer` instead of the Linux/mac location: `$HOME/.dmidiplayer`. Each configuration file is an INI file, a simple text file that you will understand easily.

**Reply by practicafz** [reply]
The Help is blank in Windows 10-64:
![drumstick-help](https://user-images.githubusercontent.com/42104122/233207299-92accd2b-a957-438a-b388-dd02fabefd7c.png)

**Reply by pedrolcl** [reply]
Converted to issue: #7
